src/qibosoq/abstracts.py

```diff
@@ -0,0 +1,65 @@
+"""Various heleper objects"""
+
+from dataclasses import dataclass
+from enum import Enum, auto
+from typing import List, Union
+
+
+@dataclass
+class Config:
+    """General RFSoC Configuration"""
+
```

Comment:
Usually, you don't put an empty line after the docstring (just conventions)
```suggestion
    """General RFSoC Configuration"""
```